superset-frontend/src/components/Label/index.tsx

```diff
@@ -34,40 +34,40 @@ export interface LabelProps {
 }
 
 const SupersetLabel = styled(BootstrapLabel)`
-  &.supersetLabel {
-    border-radius: ${({ theme }) => theme.borderRadius}px;
-    border: none;
-    color: ${({ theme }) => theme.colors.secondary.light5};
-    font-size: ${({ theme }) => theme.typography.sizes.s};
-    font-weight: ${({ theme }) => theme.typography.weights.bold};
-    min-width: ${({ theme }) => theme.gridUnit * 36}px;
-    min-height: ${({ theme }) => theme.gridUnit * 8}px;
-    text-transform: uppercase;
-    margin-left: ${({ theme }) => theme.gridUnit * 4}px;
-    &:first-of-type {
-      margin-left: 0;
-    }
-
-    i {
-      padding: 0 ${({ theme }) => theme.gridUnit * 2}px 0 0;
+  cursor: ${({ onClick }) => (onClick ? 'pointer' : 'default')};
+  &.label-warning {
+    background-color: ${({ theme }) => theme.colors.warning.base};
+    &:hover {
+      background-color: ${({ theme, onClick }) =>
+        onClick ? theme.colors.warning.dark1 : theme.colors.warning.base};
     }
-
-    &.primary {
-      background-color: ${({ theme }) => theme.colors.primary.base};
+  }
+  &.label-error {
+    background-color: ${({ theme }) => theme.colors.error.base};
+    &:hover {
+      background-color: ${({ theme, onClick }) =>
+        onClick ? theme.colors.error.dark1 : theme.colors.error.base};
     }
-    &.secondary {
-      color: ${({ theme }) => theme.colors.primary.base};
-      background-color: ${({ theme }) => theme.colors.primary.light4};
+  }
+  &.label-success {
+    background-color: ${({ theme }) => theme.colors.success.base};
+    &:hover {
+      background-color: ${({ theme, onClick }) =>
+        onClick ? theme.colors.success.dark1 : theme.colors.success.base};
     }
-    &.danger {
-      background-color: ${({ theme }) => theme.colors.error.base};
+  }
+  &.label-default {
+    background-color: ${({ theme }) => theme.colors.grayscale.base};
+    &:hover {
+      background-color: ${({ theme, onClick }) =>
+        onClick ? theme.colors.grayscale.dark1 : theme.colors.grayscale.base};
     }
   }
-
-  &.secondaryLabel {
-    background-color: ${({ theme }) => theme.colors.secondary.base};
```

Comment:
This .secondaryLabel style is being used by the card component for charts.

Reply:
see: https://github.com/preset-io/incubator-superset/blob/master/superset-frontend/src/views/CRUD/chart/ChartList.tsx#L543